test/rates_test.rb

```diff
@@ -57,6 +57,9 @@ def test_exchange_rate
 
     assert_equal 5.944602, fx.exchange_rate(:from => "AUD", :to => "HRK")
     assert_equal 0.168220, fx.exchange_rate(:from => "HRK", :to => "AUD")
+
+    assert_equal 0.00023, fx.exchange_rate(:from => 'SLL', to: 'USD')
```

Comment:
I'm using Hound on this project for a first time and still getting familiar with it also :) It is easy to configure Hound [https://houndci.com/configuration#ruby](https://houndci.com/configuration#ruby) but you don't have to bother with that, I'll do that.